apps/web-portal-client/lib/components/nav-bar/MobileMenuButton.jsx

```diff
@@ -1,16 +1,13 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import './mobileMenuButton.css'
 
 export function MobileMenuButton(props) {
   const {shape, onToggle} = props
 
   return (
     <div className="-mr-2 flex md:hidden">
-      <button
-        type="submit"
-        onClick={onToggle}
-        className="bg-gray-800 inline-flex items-center justify-center p-2 rounded-md text-gray-400 hover:text-white hover:bg-gray-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-offset-gray-800 focus:ring-white"
-      >
+      <button type="submit" onClick={onToggle} className="btn-menu">
```

Comment:
please, apply the same approach for the other components like LoginPage, Card, etc.

Reply:
I have task only for ONE component! Ha-HA!